src/en/wip-storage.md

```diff
@@ -107,29 +120,45 @@ Storage requirements _may_ be added to the 'metadata.yaml' of the charm as follo
 storage:
   data:
     type: filesystem
+    description: junk storage
+    shared: false
+    read-only: false
+    minimum-size: 100M
     location: /srv/data
 ```
 
-Here the charm is asking for storage it is calling 'data', and it further defines a type and location. It is possible to specify as many entries as desired for storage, and all of the related keys are optional. 
+Here the charm is asking for storage it is calling 'data', and it further defines a type and location. It is possible to specify as many entries as desired for storage, and all but the 'type' key are optional. The 'type' attribute specifies the type of the storage: filesystem or block (i.e. block device/disk). The 'minimum-size' attribute specifies the minimum size of the store, overriding the default of 1G if the user does not specify a size. The location specifies the path at which to mount filesystem-type storage. The 'read-only' and 'shared' attributes are currently not handled. Support will be added in a future version of Juju.
```

Comment:
We define 1G to be 1GiB. I've changed instances of 1G in descriptions to 1GiB, but left as 1G in the example CLI invocations.